packages/browser-sdk/README.md

```diff
@@ -85,6 +85,8 @@ Supply these to the constructor call (3rd argument)
     fallbackFeatures?: string[]; // Enable these features if unable to contact bucket.co
     timeoutMs?: number; // Timeout for fetching features
     staleWhileRevalidate?: boolean; // Revalidate in the background when cached features turn stale to avoid latency in the UI
+    staleTimeMs?: // at initialization time features are loaded from the cache unless they have gone stale. Defaults to 0 which means the cache is disabled. Increase in the case of a non-SPA.
```

Comment:
should we make it configurable here but keep it at 60 and instead set these to zero in react SDKs?

Reply:
great point. I went back and forth with myself on it. I think for the sake of clarity we should just set it to zero in the BrowserSDK too because people will be building SPAs with that as the basis for anything that isn't React.